docs/config-file.md

```diff
@@ -10,16 +10,25 @@ This means you can configure Unibeautify via:
 - A `unibeautify.config.js` file that exports an object.
 - A `"unibeautify"` key in your `package.json` file.
 
-The [options](options.md) to the configuration file are grouped by the respective language.
+The [options](options-for-languages.md) to the configuration file are grouped by the respective language.
```

Comment:
@szeck87 what're your thoughts on this rename? Do you think the language options should be shortened `options` and then beautifier's options be `options-for-beautifier`?

Reply:
I think it's fine what you have here.  `options-for-language` and `options-for-beautifier`.